packages/xrpl/test/integration/transactions/payment.test.ts

```diff
@@ -25,14 +30,17 @@ describe('Payment', function () {
     paymentTx = {
       TransactionType: 'Payment',
       Account: senderWallet.classicAddress,
-      Amount: AMOUNT,
+      Amount: amount,
       Destination: 'rfkE1aSy9G8Upk4JssnwBxhEv5p4mn2KTy',
     }
   })
 
   beforeAll(async () => {
     testContext = await setupClient(serverUrl)
     senderWallet = await generateFundedWallet(testContext.client)
+    // Make sure the amount sent satisfies minimum reserve requirement to fund an account.
+    amount =
```

Comment:
Hello,
The intention in these tests is not to fund an account into existence. Hence, the Amount field could be any positive value, it does not have to be above the reserve requirement.

It looks like the `amount` value is always equal to `DEFAULT_AMOUNT`. So, I don't see a need for another `amount` variable.

Are changes in this file required? I don't this file is impacted by the fee-change-calculation updates

Reply:
It is because the first payment transaction in this test `Validate Payment transaction API v2: Payment Transaction: Specify Only Amount field` will create the hardcoded account `rfkE1aSy9G8Upk4JssnwBxhEv5p4mn2KTy`, which does not exist beforehand. So if we increase `account_reserve = 20000000` instead of 10XRP then all payment transactions in these tests will fail:
`xrpl:     Message:
xrpl:       Destination does not exist. Too little XRP sent to create it.: expected 'tecNO_DST_INSUF_XRP' to equal 'tesSUCCESS'`

Hence we either have to make this first payment to be at least the account_reserve or setup a destination account before sending payments

Reply:
ah okay, thanks for the clarification